pose-detection/src/calculators/keypoints_one_euro_filter.ts

```diff
@@ -47,16 +47,28 @@ export class KeypointsOneEuroFilter implements KeypointsFilter {
     // Initialize filters once.
     this.initializeFiltersIfEmpty(keypoints);
 
+    // Get value scale as inverse value of the object scale.
+    // If value is too small smoothing will be disabled and keypoints will be
+    // returned as is.
+    let valueScale = 1;
```

Comment:
Why you remove this logic in first CL in the chain and then revert it here?
Same for all changes below.

Reply:
This change was temporary and is now gone in the newer commits